### PR TITLE
LPD-43205

### PR DESCRIPTION
**Pull request by rijema**
https://liferay.atlassian.net/browse/LPD-43205

### CODE REVIEW

**Comment by rijema** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java [hunk 1]
```diff
@@ -2507,6 +2507,10 @@ private void _testUnbindObjectDefinitionsWithObjectAction(
 
 		Assert.assertFalse(objectAction.isActive());
 
+		Assert.assertEquals(
+			objectAction.getObjectActionTriggerKey(),
+			ObjectActionTriggerConstants.KEY_ON_AFTER_UPDATE);
```
Comment (marcelabc) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java:2512]: Can you sort and remove the empty lines between the Asserts?

```
Assert.assertEquals(
	objectAction.getObjectActionTriggerKey(),
	ObjectActionTriggerConstants.KEY_ON_AFTER_UPDATE);
Assert.assertFalse(objectAction.isActive());
```

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

**Comment by rijema** [comment]
Hi @marcelabc , this removed AssertFailure is not necessary anymore, since the objectAction is switching to a non-root related one, so I removed from the test and created a single assert equals. Let me know your thoughts.

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java (hunk 1 above)
Comment (rijema) [reply modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java:2509]: Sure

**Comment by rijema** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:object

**Comment by rijema** [comment]
ci:test:sf

**Comment by rijema** [comment]
ci:forward

**Comment by rijema** [comment]
ci:forward:force

**Comment by rijema** [comment]
After reviewing the CI for objects suite and relevant one, this ticket doesn't have any unique failures anymore.

**Comment by marcelabc** [comment]
ci:forward:force

**Comment by carolmariaabb** [comment]
ci:forward:force

**Pull request closed without merging**